src/plays/play-fair-cipher/styles.css

```diff
@@ -0,0 +1,73 @@
+h1 {
+	font-size: 3rem;
+	font-weight: 700;
+	line-height: 1;
+}
+.pf-container,
+.pf-form {
+	display: grid;
+	place-items: center;
+}
+.pf-form,
+.pf-form > label {
+	margin-top: 1.5rem;
+}
+.pf-form > label {
+	font-size: 1.125rem;
+	font-weight: 700;
+	line-height: 1.75rem;
+}
+.pf-form > input[type="text"] {
+	--tw-border-opacity: 1;
+	--tw-text-opacity: 1;
+	border-color: rgb(156 163 175 / var(--tw-border-opacity));
```

Comment:
Is a TW variable necessary in here? Can't you use a regular value

Reply:
I just use tailwind generate css